save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/HasErrorModal.kt

```diff
@@ -143,7 +143,7 @@ val requestModalHandler: FC<PropsWithChildren> = FC { props ->
 
     requestStatusContext.Provider {
         value = contextPayload
-        +reactNode
+        +props.children
```

Comment:
Are changes in this file intentional?

Reply:
Yes, the page does not load without them.
Vlad fixed it, but apparently I shouldn't have commit it